Templates/RemoteRepo/Template/repo.json

```
@@ -0,0 +1,13 @@
+{
+    "repo_name":"${Name}",
+    "repo_uri": "${RepoURI}",
+    "repo_path": "${RepoPath}",
+    "schemaVersion": "1.0.0",
+    "origin":"Origin for the ${Name} Gem Repository",
+    "summary": "${Summary}",
+    "additional_info": "Additional info for ${Name}",
+    "last_updated": "",
+    "gems_data": [],
+    "projects_data": [],
+    "template_data":[]
```

Missing `s` in `templates_data`
```suggestion
    "templates_data":[]
```